tests/pytest_tests/system_tests/test_launch/test_job.py

```diff
@@ -43,7 +43,9 @@ def test_job_call(relay_server, user, wandb_init, test_settings):
         run.finish()
 
 
+@pytest.mark.wandb_core_failure(feature="launch")
```

Comment:
why is it failing?

Reply:
This is failing because this test explicitly checks for the behavior we are trying to adjust in the core job builder (but not the python job builder). I will leave a comment here and add a system test for the behavior we want in the core job builder.

Reply:
After another review with @KyleGoyette we ended up modifying the behavior in both job builders. I've modified the test to check the new behavior and it will run for core and no-core.